### PR TITLE
Issue #135

### PR DESCRIPTION
**Pull request by pahnin**
Fixed issue 135 and notification on failure of ajax get request in home page

### CODE REVIEW

File: app/views/home/_group_projects.html.haml~ [hunk 1]
```diff
@@ -0,0 +1,16 @@
```
Comment (srushti) [inline app/views/home/_group_projects.html.haml~:0]: Please fix your global .gitignore and/or editor. Your editor seems to be generating these '~' files (there's two of them in this pull request). Also, delete these files from this pull request.

File: public/stylesheets/reset.css [hunk 2]
```diff
@@ -1,156 +0,0 @@
```
Comment (srushti) [inline public/stylesheets/reset.css]: This isn't getting generated from anything. We just got this from somewhere else. Our .css files are being generated by us, but not this one. It needs to be restored.

Comment (pahnin) [inline public/stylesheets/reset.css]: I think I removed all the css files in the stylesheets directory! So, whats my best option to restore it?

Comment (pahnin) [inline public/stylesheets/reset.css]: I did the recovery and deleted the ~ files, closing this pull- request and re requesting.
Thanks

**Pull request closed without merging**